bigquery_etl/view/__init__.py

```diff
@@ -194,7 +194,10 @@ def publish(self, target_project=None, dry_run=False):
 
         # avoid checking references since Jenkins might throw an exception:
         # https://github.com/mozilla/bigquery-etl/issues/2246
-        if self._valid_view_naming():
+        if (
+            any(str(self.path).endswith(p) for p in SKIP_VALIDATION)
```

Comment:
We'll still need to skip views here, like `sql/moz-fx-data-shared-prod/telemetry/client_probe_counts_v1/view.sql`